cmd/podman/commit.go

```diff
@@ -90,20 +94,19 @@ func commitCmd(c *cli.Context) error {
 		History: history,
 		Author:  c.String("author"),
 	}
-	opts.ImageConfig = config
-	opts.Writer = nil
+	writer = nil
```

Comment:
Not necessary, writer is already initialized to nil

Reply:
fixed